src/components/icon/Icon.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
Is it '@fort...' or '@font...' in '@fortawesome/react-fontawesome';

Reply:
It is fort. I wonder the same why is it so but it is.